src/main/java/org/fundacionjala/sfdc/util/PropertiesManager.java

```diff
@@ -12,6 +14,7 @@
 public final class PropertiesManager {
     private static PropertiesManager propertiesManager;
     private Properties prop;
+    private static final Logger log = Logger.getLogger(PropertiesManager.class.getName());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename the "log" logger to comply with the format "LOG(?:GER)?". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1312)